yarn.lock

```diff
@@ -1,2404 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
please use `npm ci` instead of yarn for this repo. this way the package.json.lock stays the single source of truth for the package versions

Reply:
done

Reply:
I think you still need to undo the prev changes (removing the yarn.lock file)